mesonbuild/coredata.py

```diff
@@ -1216,6 +1216,7 @@ def add_to_argparse(self, name: str, parser: argparse.ArgumentParser, help_suffi
     (OptionKey('werror'),          BuiltinOption(UserBooleanOption, 'Treat warnings as errors', False, yielding=False)),
     (OptionKey('wrap_mode'),       BuiltinOption(UserComboOption, 'Wrap mode', 'default', choices=['default', 'nofallback', 'nodownload', 'forcefallback', 'nopromote'])),
     (OptionKey('force_fallback_for'), BuiltinOption(UserArrayOption, 'Force fallback for those subprojects', [])),
+    (OptionKey('dub_fetch_build'), BuiltinOption(UserBooleanOption, 'Whether to fetch and build the missing DUB dependencies', True)),
```

Comment:
Maybe there should be a dub module? Would hate to put dub_ in the top-level options

Reply:
More of a complaint really.

Reply:
There is a `dlang` module.

Reply:
There is already a dlang module that generate dub.json files. But the dub dependencies do not depend on it.
You mean that the module would provide a function to download and fetch missing libraries?

Reply:
I am not super familiar with how module options work, but I would think you could read that option in the regular dependency() call like you are doing right now. @eli-schwartz knows better about module-specific options.

Reply:
module-specific options are only shown in configure, if that module has been `import()`ed. But otherwise they exist just fine and can be in theory used for any purpose, though one might debate whether it "should" be used for code which is not resident inside said module.

Reply:
Probably fine as-is